zio-http/src/test/scala/zhttp/http/StatusSpec.scala

```diff
@@ -0,0 +1,29 @@
+package zhttp.http
+
+import zhttp.internal.HttpMessageAssertions
+import zhttp.service.EventLoopGroup
+import zio.test.Assertion._
+import zio.test._
+
+object StatusSpec extends DefaultRunnableSpec with HttpMessageAssertions {
+  private val env = EventLoopGroup.auto(1)
```

Comment:
No need to use a real server for these tests. An HttpApp can simply be called like a function.

```scala
val app : HttpApp[_, _] = ???
app(Request()) // ZIO[_, _, Response]
```
Something like above.

Reply:
Oh, that is really good to know!